components/Board.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Letter from "./Letter";
+
+export default function Board() {
+  return (
+    <div className="board">
+      {" "}
```

Comment:
**ease of understanding/fitness for purpose**: what is the reason for having this space here? Does the div require text content within it? This seems extraneous to me